### ISSUE TITLE
Fouc Orbit 6

### ISSUE THREAD

**Issue by Joseluisblom** [zurb/foundation-sites#8828]
hi, i can see each image above others for a brief second when i reload my website using orbit, i'm using the same code of foundation docs, this: http://foundation.zurb.com/sites/docs/orbit.html

how can i avoid fouc in orbit? regards

**Comment by informaticphp** [comment]
btw i'm using Flex grid

**Comment by Joseluisblom** [comment]
btw i am using Flex-grid

**Comment by Joseluisblom** [comment]
please somone there? still can't solve the problem : (

**Comment by rediris** [comment]
My solution was to add inline CSS to the non-active slides.

```
<li class="is-active orbit-slide">
  <img class="orbit-image" src="assets/img/orbit/01.jpg" alt="Space">
  <figcaption class="orbit-caption">Space, the final frontier.</figcaption>
</li>
<li class="orbit-slide" style="display: none;">
  <img class="orbit-image" src="assets/img/orbit/02.jpg" alt="Space">
  <figcaption class="orbit-caption">Lets Rocket!</figcaption>
</li>
<li class="orbit-slide" style="display: none;">
  <img class="orbit-image" src="assets/img/orbit/03.jpg" alt="Space">
  <figcaption class="orbit-caption">Encapsulating</figcaption>
</li>
<li class="orbit-slide" style="display: none;">
  <img class="orbit-image" src="assets/img/orbit/04.jpg" alt="Space">
  <figcaption class="orbit-caption">Outta This World</figcaption>
</li>
```

**Comment by andycochran** [comment]
You might also try: 
```scss
.orbit-slide:not(.is-active) {
  display: none; 
}
```
Or maybe a `.no-js { ... }` approach.

**Comment by andycochran** [comment]
It would be nice if Orbit could switch classes on elements instead of applying inline styles.

**Comment by kball** [comment]
I believe the FOUC issue has been resolved by https://github.com/zurb/foundation-sites/pull/9317

**Comment by maclein** [comment]
@kball using the latest version of foundation. the FOUC issue on the orbit plugin has not been resolved by #9317

**Comment by kball** [comment]
ok reopening based on @maclein's comment. @maclein can you provide an example?
